consensus/spos/constants.go

```diff
@@ -2,3 +2,19 @@ package spos
 
 // MaxThresholdPercent specifies the max allocated time percent for doing Job as a percentage of the total time of one round
 const MaxThresholdPercent = 95
+
+// ProposerRatingIncreaseFactor specifies the factor with which the rating of a proposer should be increased if it
+// proposed a block or sent final info, in its correct allocated slot/time-frame/round
+const ProposerRatingIncreaseFactor = 2.0
```

Comment:
ProposerPeerQualityIncreaseFactor?   rating->PeerQuality as to avoid confusions with peer rating mechanism

Reply:
ProposerHonestyIncreaseFactor?